### DISCUSSION TITLE
ToC and Tab

### DISCUSSION THREAD

**Discussion by kaitoukid-1412** [facebook/docusaurus#8096] [Ideas]
Hi,

In single tab, with `same headings` but `different content`. ToC generated all headings in all tabs.

![image](https://user-images.githubusercontent.com/15698245/190082867-667dd523-eb40-462c-89e9-50de2f41ed5c.png)
![image](https://user-images.githubusercontent.com/15698245/190082897-23a8994e-18ee-43f9-94e8-9a2e3430f772.png)
![image](https://user-images.githubusercontent.com/15698245/190083042-9359b6f1-ff0e-45de-bb80-772cae8a97f6.png)

I think each tab just shows its own `ToC`

Like this:
![image](https://user-images.githubusercontent.com/15698245/190085001-bc598aac-f769-409a-bce7-a44e4b88340e.png)
![image](https://user-images.githubusercontent.com/15698245/190085100-fcb024bc-0fe8-4a61-a81c-687014f76262.png)
![image](https://user-images.githubusercontent.com/15698245/190085176-abe0b8aa-c3c7-4e3b-8e94-5a4624f986d5.png)

To:
- avoid duplicating the headings
- avoid some links not working if `tab is not active`

Demo:

```
https://stackblitz.com/edit/github-hb3gwh?file=docs/tab.mdx
```

**Comment by slorber** [comment]
Hey

I'm not 100% sure to understand what you mean, but I guess you'd like to see this feature being implemented: https://github.com/facebook/docusaurus/pull/946#issuecomment-1228648367

For now we don't support nesting headings inside tabs
